### PR TITLE
[16.0][MIG] partner_sale_pivot: Migration to 16.0

### PR DESCRIPTION
**Pull request by KM-joel**
Migration to version 16.0

 - partner_sale_pivot

I removed the groups in the inherit view sale.res_partner view_buttons because the inherited view cannot have "groups" defined in the record.

### CODE REVIEW

File: partner_sale_pivot/views/res_partner_views.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
         <field name="name">res.partner.form.sales_by_product</field>
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="sale.res_partner_view_buttons" />
-        <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]" />
```
Comment (pedrobaeza) [inline partner_sale_pivot/views/res_partner_views.xml]: Put the group on the button itself.

Comment (KM-joel) [reply partner_sale_pivot/views/res_partner_views.xml]: Hey @pedrobaeza ,

if the group already exists in the button.

**Review by pedrobaeza — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by KM-joel** [comment]
Hey @ernestotejeda @victoralmau ,

Request review.

**Comment by pedrobaeza** [comment]
@kjoeldevk this is not how it goes. You should review other PRs and ask in exchange that they review yours.

**Comment by KM-joel** [comment]
> @kjoeldevkce n'est pas comme ça que ça se passe. Vous devriez passer en revue les autres PR et demander en échange qu'ils passent en revue le vôtre.

@pedrobaeza 

ok thanks i am new.

**Review by bealdav — APPROVED** [review]
LGTM

**Pull request merged**